config.json

```diff
@@ -77,7 +80,7 @@
           "viewName": "Extraction Path",
           "description": "Path to directory into which the zip file will be extracted",
           "type": "string",
-          "required": true,
```

Comment:
This change will break the funtionality. If you really want to turn this parameter non-required, then we need to adjust the code accordingly.